php-checks/src/test/resources/checks/regex/DuplicatesInCharacterClassCheck.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace checks\regex;
+
+class DuplicatesInCharacterClassCheck
+{
+
+  function noncompliant($input)
+  {
+    preg_match("/[0-99]/", $input); // Noncompliant {{Remove duplicates in this character class.}}
+    preg_match("/[90-9]/", $input); // Noncompliant
+    preg_match("/[0-73-9]/", $input); // Noncompliant
+    preg_match("/[0-93-57]/", $input); // Noncompliant
+    preg_match("/[4-92-68]/", $input); // Noncompliant
+    preg_match("/[0-33-9]/", $input); // Noncompliant
+    preg_match("/[0-70-9]/", $input); // Noncompliant
+    preg_match("/[3-90-7]/", $input); // Noncompliant
+    preg_match("/[3-50-9]/", $input); // Noncompliant
+    preg_match("/[xxx]/", $input); // Noncompliant
+    preg_match("/[A-z_]/", $input); // Noncompliant
+    preg_match("/(?i)[A-Za-z]/", $input); // Noncompliant
```

Comment:
Maybe it's worth adding a false-negative example of when the flag is set after the closing delimiter (I think this is the most common way in PHP) `/aA/i`. Not sure if there is a ticket to support that.